pagopa/io-app
# feat(Firma con IO): [SFEQS-1460] Add signature requests list screen

## Short description
This PR adds a new screen to receive signature requests from the current SIGNER. This screen is available via internalLink in the FCI FAQ. Used to allow the user to request assistance.

https://user-images.githubusercontent.com/49342144/232020917-1865e016-f31d-4eac-9143-2f0e97ee5fa4.mp4

## List of changes proposed in this pull request
- Add SignatureRequestItem.tsx
- Add FciSignatureRequestScreen.tsx

## How to test
The fastest way to try this feature is to edit the FciStackNavigator.tsx as below. So tapping on every FCI message the list of signature requests should be rendered.

```typescript
//...
export const fciLinkingOptions: PathConfigMap = {
  [FCI_ROUTES.MAIN]: {
    path: "fci",
    screens: {
      [FCI_ROUTES.SIGNATURE_REQUESTS]: "main"
      // [FCI_ROUTES.SIGNATURE_REQUESTS]: "signature-requests"
    }
  }
};
//...
```
